services/QuillLMS/app/queries/progress_reports/standards/new_student.rb

```diff
@@ -98,8 +98,8 @@ def results(filters)
   end
 
   private def classroom_conditional
-    if @classroom_id && @classroom_id != 0 && @classroom_id != '0'
-      "AND classrooms.id = #{@classroom_id}"
-    end
+    return unless @classroom_id && @classroom_id != 0 && @classroom_id != '0'
```

Comment:
```suggestion
    return unless @classroom_id 
    return if @classroom_id.in?([0,'0'])
```

Reply:
I'm going to use `include?` since I think it's [faster](https://stackoverflow.com/a/28272687)